src/main/java/com/app/backend/model/Decedent.java

```diff
@@ -0,0 +1,55 @@
+package com.app.backend.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+import java.time.Period;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "_decedent")
+public class Decedent {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
+    @Column(unique = true, nullable = false)
+
+    private String name;
+    private String surname;
+    @Column(nullable = false)
+
+    private LocalDate birthDate;
+    private LocalDate deathDate;
+
+    private String description;
+
+    private float latitude;
+    private float longitude;
+
+    @ManyToOne
+    @JoinColumn(name = "cemetery_id")
+    private Cemetery cemetery;
+
+    @Lob
+    @Column(name = "tombstone_image")
+    private byte[] tombstoneImage;
+
+    @ManyToOne
+    @JoinColumn(name="user_id")
+    private User user;
+
+    public int calculateAge(){
```

Comment:
You can add here //todo for example as we don’t use this method so it’s good to add todo.